quiz/generic/quiz_page.inc

```diff
@@ -215,8 +215,8 @@ function qp_text_contains_anticipated_error($text, $test)
                 $part = strstr($text, $test["starttext"]);
                 $part = substr($part, $len, strpos($part, $test["stoptext"]) - $len);
             } else {
-                $part = stristr($text, $test["starttext"]);
-                $part = substr($part, $len, stripos($part, $test["stoptext"]) - $len);
+                $part = voku\helper\UTF8::stristr($text, $test["starttext"]);
```

Comment:
Why not use mb_stristr here? Wouldn't it be better to use a php function rather than a 3rd party one?
Simiarly stripos etc.

Reply:
We could, but we've standardized on using the UTF8 package for all our multibyte work rather than `mb_*` calls (or `iconv` calls) directly and needing to provide the string encoding (which is always UTF-8). This was an conscious decision during the Unicode conversion.

Under the covers the UTF8 package is doing the necessary `mb_*` or `iconv` or other calls but it provides a consistent interface.